includes/woopay/class-woopay-adapted-extensions.php

```diff
@@ -19,6 +19,9 @@ class WooPay_Adapted_Extensions extends IntegrationRegistry {
 	const GIFT_CARDS_API            = 'woocommerce-gift-cards';
 	const GIFT_CARDS_BLOCKS         = 'wc-gift-cards-blocks';
 
+	const WOOCOMMERCE_MULTICURRENCY      = 'woocommerce-multicurrency';
+	const WOOCOMMERCE_MULTICURRENCY_PATH = 'woocommerce-multicurrency/woocommerce-multicurrency.php';
```

Comment:
Should we check if a constant in the plugin is defined so if the folder name is different then we can prevent critical error?

Reply:
Good catch, fixed on https://github.com/Automattic/woocommerce-payments/pull/7246/commits/f3c67b1c24d42bd0496f94bafd753fd8b681b213.